diff --git a/node_modules/gotrue-js/lib/user.js b/node_modules/gotrue-js/lib/user.js
index 53587ae..b7e0865 100644
--- a/node_modules/gotrue-js/lib/user.js
+++ b/node_modules/gotrue-js/lib/user.js
@@ -284,6 +284,14 @@ var User = /*#__PURE__*/function () {
 
 exports["default"] = User;
 
+function atob(string) {
+  if(isBrowser()) {
+    return window.atob(string)
+  } else {
+    return Buffer.from(string, 'base64').toString()
+  }
+}
+
 function urlBase64Decode(str) {
   // From https://jwt.io/js/jwt.js
   var output = str.replace(/-/g, '+').replace(/_/g, '/');
@@ -304,8 +312,7 @@ function urlBase64Decode(str) {
       throw 'Illegal base64url string!';
   } // polifyll https://github.com/davidchambers/Base64.js
 
-
-  var result = window.atob(output);
+  var result = atob(output)
 
   try {
     return decodeURIComponent(escape(result));
